examples/examples/data-channels-flow-control/data-channels-flow-control.rs

```diff
@@ -1,368 +1,263 @@
+                })
             }))
             .await;
 
-            // Register the OnMessage to handle incoming messages
-            dc.on_message(Box::new(move |msg: DataChannelMessage| {
-                let n = msg.data.len();
-                total_bytes_received.fetch_add(n, Ordering::SeqCst);
-                Box::pin(async {})
+            dc.on_message(Box::new(move |msg| {
+                let total_bytes_received = total_bytes_received.clone();
+
+                Box::pin(async move {
+                    total_bytes_received.fetch_add(msg.data.len(), Ordering::Relaxed);
+                })
             }))
             .await;
         })
     }))
     .await;
 
-    Ok(pc)
+    pc
 }
 
 #[tokio::main]
-async fn main() -> Result<()> {
-    let mut app = Command::new("data-channels-flow-control")
-        .version("0.1.0")
-        .author("Rain Liu <yliu@webrtc.rs>")
-        .about("An example of Data-Channels-Flow-Control.")
-        .setting(AppSettings::DeriveDisplayOrder)
-        .subcommand_negates_reqs(true)
-        .arg(
-            Arg::new("FULLHELP")
-                .help("Prints more detailed help information")
-                .long("fullhelp"),
-        )
-        .arg(
-            Arg::new("debug")
-                .long("debug")
-                .short('d')
-                .help("Prints debug log information"),
-        );
-
-    let matches = app.clone().get_matches();
-
-    if matches.is_present("FULLHELP") {
-        app.print_long_help().unwrap();
-        std::process::exit(0);
-    }
-
-    let debug = matches.is_present("debug");
-    if debug {
-        env_logger::Builder::new()
-            .format(|buf, record| {
-                writeln!(
-                    buf,
-                    "{}:{} [{}] {} - {}",
-                    record.file().unwrap_or("unknown"),
-                    record.line().unwrap_or(0),
-                    record.level(),
-                    chrono::Local::now().format("%H:%M:%S.%6f"),
-                    record.args()
-                )
-            })
-            .filter(None, log::LevelFilter::Trace)
-            .init();
-    }
-
-    {
-        let offer_pc = create_offerer().await?;
-        let answer_pc = create_answerer().await?;
-
-        // Set ICE Candidate handler. As soon as a PeerConnection has gathered a candidate
-        // send it to the other peer
-        let offer_pc2 = Arc::downgrade(&offer_pc);
-        answer_pc
-            .on_ice_candidate(Box::new(move |c: Option<RTCIceCandidate>| {
-                let offer_pc3 = offer_pc2.clone();
-                Box::pin(async move {
-                    if let Some(c) = c {
-                        if let Ok(c) = c.to_json().await {
-                            if let Some(offer_pc3) = offer_pc3.upgrade() {
-                                let _ = offer_pc3.add_ice_candidate(c).await;
-                            }
+async fn main() {
+    let requester = Arc::new(create_requester().await);
```

Comment:
I know the help menu here wasn't the most expansive, but the convention is that all examples have one. Please reintroduce it